### PR TITLE
app-backup/burp: fix strptime for musl

### PR DESCRIPTION
**Pull request by antonfischl1980**
Closes: https://bugs.gentoo.org/862019

Signed-off-by: Anton Fischl <github@fischl-online.de>

### CODE REVIEW

**Comment by antonfischl1980** [comment]
I don't think a revision bump is necessary. On glibc nothing changed and on musl it should have never worked up until now anyway.

**Comment by thesamesam** [comment]
> I don't think a revision bump is necessary. On glibc nothing changed and on musl it should have never worked up until now anyway.

Users could have built it but it would be broken at runtime.

**Comment by antonfischl1980** [comment]
> Users could have built it but it would be broken at runtime.

Ah, yeah, I see your point. I changed it now to a revision bump.

File: app-backup/burp/files/burp-2.4.0-fix-musl-strptime.patch [hunk 1]
```diff
@@ -0,0 +1,82 @@
+From 76b7f1ba9f4445108059b13f3d79a7fde8a292a3 Mon Sep 17 00:00:00 2001
```
Comment (thesamesam) [inline app-backup/burp/files/burp-2.4.0-fix-musl-strptime.patch:1]: I'm going to throw in a link to the bug & commit: https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto.

**Comment by thesamesam** [comment]
Good work, thanks!

**Pull request closed without merging**